menas/ui/components/schema/schemaDetail.controller.js

```diff
@@ -172,8 +172,11 @@ sap.ui.define([
 
       if (status === 500) {
         const sSchemaType = this.byId("schemaFormatSelect").getSelectedItem().getText();
+        let oRawResponse = JSON.parse(oParams.getParameter("responseRaw"));
+        const errorMessage = oRawResponse.message;
```

Comment:
Why `const`?

Reply:
Var vs Val :)

I'd say oRawResponse should be const too...

Reply:
OK, it seemed little against our current style, when basically everywhere we use `let`. Also especially for a local variable in JS, it seems uncommon, but if it's the new way, I have no issues going with it 😉

Reply:
`const` is completely valid in a local scope. And the reason our style so far has been mostly `let` is because I was slow to adopt `const`.